spec/spec_helper.rb

```diff
@@ -99,11 +99,18 @@
     ActionMailer::Base.deliveries = []
   end
 
+  # This is a hack to get around the ssl_required? method in
```

Comment:
@henare is this what you were imagining https://github.com/openaustralia/planningalerts/commit/e45a180d6a28e72908855a9b85f1bb8c4c2da6bf ?